precompiles/parachain-staking/src/lib.rs

```diff
@@ -95,14 +95,14 @@ where
 			}
 
 			// If not an accessor, check for dispatchables. These calls ready for dispatch below.
-			[0xad, 0x76, 0xed, 0x5a] => Self::join_candidates(&input[SELECTOR_SIZE_BYTES..])?,
-			[0xb7, 0x69, 0x42, 0x19] => Self::leave_candidates(&input[SELECTOR_SIZE_BYTES..])?,
+			[0x0a, 0x1b, 0xff, 0x60] => Self::join_candidates(&input[SELECTOR_SIZE_BYTES..])?,
```

Comment:
```suggestion
			&Keccak256::digest("join_candidates(uint256,uint256") => Self::join_candidates(&input[SELECTOR_SIZE_BYTES..])?,
```
This makes the code more readable IMO

Reply:
We already test this as you commented above. I expect this suggestion to incur some performance hit. Every time the staking precompile interface is used, we'd be hashing all of these until it matches on the right one.